cdap-docs/build.sh

```diff
@@ -38,227 +66,197 @@ function set_project_path() {
   fi
 }
 
-function check_starting_directory() {
+function setup() {
+  # Check that we're starting in the correct directory
+  local quiet={$1}
   E_WRONG_DIRECTORY=85
-
   if [[ "x${MANUAL}" == "x" || "x${CDAP_DOCS}" == "x" ]]; then
     echo "Manual or CDAP_DOCS set incorrectly: are you in the correct directory?"
     exit ${E_WRONG_DIRECTORY}
   fi
-  
   if [[ " ${MANUALS[@]}" =~ "${MANUAL} " || "${MANUAL}" == "${CDAP_DOCS}" ]]; then
-    echo "Using \"${MANUAL}\""
+    if [[ "x${quiet}" == "x" ]]; then
+      echo "Check for starting directory: Using \"${MANUAL}\""
+    fi
+    set_project_path
     return 0
   else  
     echo "Did not find MANUAL \"${MANUAL}\": are you in the correct directory?"
     exit ${E_WRONG_DIRECTORY}
   fi
-  
   exit 1
 }
 
-function usage() {
-  cd ${PROJECT_PATH}
-  PROJECT_PATH=`pwd`
-  echo "Build script for '${PROJECT_CAPS}' docs"
-  echo "Usage: ${SCRIPT} < option > [source test_includes]"
-  echo ""
-  echo "  Options (select one)"
-  echo "    all            Clean build of everything: HTML docs and Javadocs, GitHub and Web versions"
-  echo ""
-  echo "    doc            Clean build of just the outer level HTML docs, skipping Javadocs and zipping"
-  echo "    docs           Clean build of just the HTML docs, skipping Javadocs, zipped for placing on docs.cask.co webserver"
-  echo "    docs-github    Clean build of HTML docs and Javadocs, zipped for placing on GitHub"
-  echo "    docs-web       Clean build of HTML docs and Javadocs, zipped for placing on docs.cask.co webserver"
-  echo ""
-  echo "    javadocs       Build Javadocs"
-  echo "    licenses       Clean build of License Dependency PDFs"
-  echo "    sdk            Build SDK"
-  echo "    version        Print the version information"
-  echo ""
-  echo "    clean          Clean up (previous builds)"
-  echo ""
-  echo "  with"
-  echo "    source         Path to ${PROJECT} source, if not ${PROJECT_PATH}"
-  echo "    test_includes  local, remote or neither (default: remote); must specify source if used"
-  echo ""
-}
-
 function run_command() {
   case "${1}" in
```

Comment:
No need to quote. Best practice is to be conservative with quotes and only use them where necessary.